### DISCUSSION TITLE
x-bind:value on radio input not updating value of element when bound x-data property changes

### DISCUSSION THREAD

**Discussion by megangessner** [alpinejs/alpine#3760] [1. Help]
Hello there! 

I'm a huge fan of alpine, love the expressivity and clarity it provides me. 

I've made a really strange observation, and I'm unsure if it's an alpine bug or perhaps a quirk of inputs. Basically, if I bind a data property to the value of a radio input, the only time the input value gets updated is the very first time it gets set, never when the bound data changes. But, if I do a `x-bind:value` on a `input type="hidden"`, the hidden input's value does change with the change in the associated x-data property value. 

Here's a code pen demonstrating the issue:
https://codepen.io/megan-gessner/pen/wvRejJN

By default all the values are initialized to the x-data's number property, but as you change number, only the hidden input's value responds to the change.

For context, I'm using a recent version of alpine (3.4.2).

I've found this problem extends to text-inputs as well. But that's less relevant to my particular use-case :) 

Anyone know what's going on here?

**Comment by ekwoka** [comment]
Well, start by updating Alpine to 3.13.

It doesn't actually change anything here, but it's better.

This is a bit strange, I'd bet it's something to do with how radio aren't really meant to have their values changed.
